Header header
uint32 keepAlive32request
uint32 keepAlive32answer
